README.md

```diff
@@ -1,20 +1,20 @@
 # cloud.gov Demos
 
-This project provides several live interactive demonstrations of 
-cloud.gov functionality.
+This project provides several a interactive demonstration of
+Cloud.gov functionality.
 
 ## Useful links for the demo:
 
-* [cloud.gov login](https://login.fr.cloud.gov/login)
-* [cloud.gov dashboard](https://dashboard-beta.fr.cloud.gov)
-* [cf-sample-app-spring repository](https://github.com/18F/cf-sample-app-spring)
+* [Cloud.gov login](https://login.fr.cloud.gov/login)
+* [Cloud.gov dashboard](https://dashboard.fr.cloud.gov)
+* [sample-app repository](https://github.com/pburkholder/sample-app)
```

Comment:
Do we have this sample-app in the `cloud-gov` org? We probably should if we're going to reference it from here? Can we use something from https://github.com/cloud-gov/cf-hello-worlds to avoid repo sprawl?

Reply:
I use sample-app because as a Go app it stages really fast, and it provides the feedback we want on app status and connected DBs. But, yes, switched to a cloud-gov fork.